mitosheet/src/components/taskpanes/SplitTextToColumns/SplitTextToColumnsTaskpane.tsx

```diff
@@ -0,0 +1,315 @@
+                                    }
+                                    return {
+                                        ...prevParams,
+                                        delimiters: newDelimiters
+                                    }
+                                })
+                            }}
+                        />
+                    </Col>
+                </Row>
+                <div>
+                    <Row>
+                        <p className='text-header-3'>
+                            Columns Preview
+                        </p>
+                    </Row>
+                    {preview !== undefined &&
+                        <Row style={{width: '100%', overflowX: 'auto'}} suppressTopBottomMargin>
+                            <table className="preview-table" cellSpacing="0">
+                                <tbody>
+                                    {preview.map((rowData, idx) => {
+                                        return (
+                                            <tr className='preview-table-table-row' key={idx}>
+                                                {rowData.map((cellData, idx) => {
+                                                    return (
+                                                        <td className='preview-table-table-data' key={idx}>{'' + cellData}</td>
+                                                    )
+                                                })}
+                                            </tr>
+                                        )
+                                    })}
+                                </tbody>
+                            </table>
+                        </Row>
+                    }
+                    {preview === undefined && 
+                        <Row style={{width: '100%'}} justify='center'>
+                            <p className='mt-10px'>
+                                Select a delimiter to preview the split
+                            </p>
+                        </Row>
+                    }
+                </div>
+            </DefaultTaskpaneBody>
+            <DefaultTaskpaneFooter>
+                <TextButton
+                    variant='dark'
+                    width='block'
+                    onClick={() => edit(prevParams => {
+                        return {
+                            ...prevParams, 
+                            // Prepare for the user splitting again by creating a new column suffix,
+                            // so that the column headers don't overlap
+                            new_column_header_suffix: getNewColumnHeader()
+                        }
+                    })}
+                    disabled={params.delimiters.length === 0}
+                    disabledTooltip="Select at least one delimiter"
+                >
+                    {!editApplied 
+                        ? 'Split on delimiter'
```

Comment:
Can this (and the text below) say "Split on delimiter(s)" - like add an s when there are multiple?